src/containers/TasksByDayPage/components/TabContainer/index.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+import Typography from '@material-ui/core/Typography'
+import styled from 'styled-components'
+
+export default function TabContainer(props) {
```

Comment:
```suggestion
export default function TabContainer({ children }) {
```
desconstruct luôn ở đây (ES6(